msg/tools/uorb_rtps_message_ids.yaml

```
@@ -246,36 +246,54 @@ rtps:
   - msg: collision_constraints
     id: 107
     send: true
+  - msg: orbit_status
+    id: 108
+  - msg: power_monitor
+    id: 109
+  - msg: landing_gear
+    id: 110
   # multi topics
-  - msg: actuator_control0
+  - msg: actuator_controls_0
     id: 120
-  - msg: actuator_control1
+    alias: actuactor_controls
```

Typo in a bunch of these below
```suggestion
    alias: actuator_controls
```

---

Ups! That got passed. Fixed and added a check here: https://github.com/PX4/Firmware/pull/12137/commits/d231e948b085a53c2846e0805379886f86afc108